bin/pkg.ml

```diff
@@ -320,8 +320,17 @@ module Lock = struct
         ]
   ;;
 
+  let xdg_repo_location =
```

Comment:
I think this already exists in `opam_repo`, so I would suggest exporting it out of there and using here.